command/server/config.go

```diff
@@ -905,3 +905,128 @@ func parseTelemetry(result *Config, list *ast.ObjectList) error {
 
 	return nil
 }
+
+// Sanitized returns a copy of the config with all values that are considered
+// sensitive stripped. It also strips all raw values that are mainly
+// used for parsing.
+//
+// Specifically, the fields that this method strips are:
+// - Storage.Config
+// - HAStorage.Config
+// - Seals.Config
+// - Telemetry.CirconusAPIToken
+func (c *Config) Sanitized() map[string]interface{} {
+	result := map[string]interface{}{
+		"CacheSize":             c.CacheSize,
```

Comment:
Since these are ultimately going to be json keys should they be formatted like `cache_size`?

Reply:
Ah yep, good catch.

Reply:
Should be addressed in da85f38.